### ISSUE TITLE
rpi 4b aarch64 build error

### ISSUE THREAD

**Issue by genius0182** [koenvervloesem/tensorflow-addons-on-arm#3]
Build configurations successfully written to .bazelrc

Starting local Bazel server and connecting to it...
WARNING: /root/.cache/bazel/_bazel_root/05fec5a36c83f24c9f73cdaccf3bbd7b/external/local_config_tf/BUILD:2012:1: target '_pywrap_tensorflow_internal.so' is both a rule and a file; please choose another name for the rule
INFO: Analyzed target //:build_pip_pkg (37 packages loaded, 239 targets configured).
INFO: Found 1 target...
INFO: Deleting stale sandbox base /root/.cache/bazel/_bazel_root/05fec5a36c83f24c9f73cdaccf3bbd7b/sandbox
ERROR: /opt/build_soft/tensorflow-addons-on-arm/addons-0.7.1/tensorflow_addons/custom_ops/layers/BUILD:7:1: Linking of rule '//tensorflow_addons/custom_ops/layers:_correlation_cost_ops.so' failed (Exit 1)
/usr/bin/ld.gold: error: cannot find -l_pywrap_tensorflow_internal
collect2: error: ld returned 1 exit status
Target //:build_pip_pkg failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 24.981s, Critical Path: 8.54s
INFO: 0 processes.
FAILED: Build did NOT complete successfully

**Comment by koenvervloesem** [comment]
That's understandable, as TensorFlow Addons on ARM (from the README's introduction) "builds TensorFlow Addons for the Raspberry Pi's ARMv7 processor architecture" and you need a 32-bit Raspberry Pi OS for it. You seem to be running it on 64-bit Raspberry Pi OS?

I actually do want to try to build it on aarch64 some time, but I'm not running any 64-bit Raspberry Pi OS machines at the moment, so it won't happen soon.

**Comment by genius0182** [comment]
Yes,I running it on 64-bit Raspberry Pi OS. Do you 64-bit need to change anything?

**Comment by koenvervloesem** [comment]
I just haven't looked yet at building it for 64-bit ARM. You should find some help in the repository of TensorFlow Addons: https://github.com/tensorflow/addons

**Comment by genius0182** [comment]
Inquiries are already being made. Thank you very much.

**Comment by genius0182** [comment]
I used "ld -l_pywrap_tensorflow_internal --verbose" command  to find the following file failed to open,an error is as follows
`
==================================================
attempt to open //usr/local/lib/aarch64-linux-gnu/lib_pywrap_tensorflow_internal.so failed
attempt to open //usr/local/lib/aarch64-linux-gnu/lib_pywrap_tensorflow_internal.a failed
attempt to open //lib/aarch64-linux-gnu/lib_pywrap_tensorflow_internal.so failed
attempt to open //lib/aarch64-linux-gnu/lib_pywrap_tensorflow_internal.a failed
attempt to open //usr/lib/aarch64-linux-gnu/lib_pywrap_tensorflow_internal.so failed
attempt to open //usr/lib/aarch64-linux-gnu/lib_pywrap_tensorflow_internal.a failed
attempt to open //usr/local/lib/lib_pywrap_tensorflow_internal.so failed
attempt to open //usr/local/lib/lib_pywrap_tensorflow_internal.a failed
attempt to open //lib/lib_pywrap_tensorflow_internal.so failed
attempt to open //lib/lib_pywrap_tensorflow_internal.a failed
attempt to open //usr/lib/lib_pywrap_tensorflow_internal.so failed
attempt to open //usr/lib/lib_pywrap_tensorflow_internal.a failed
attempt to open //usr/aarch64-linux-gnu/lib/lib_pywrap_tensorflow_internal.so failed
attempt to open //usr/aarch64-linux-gnu/lib/lib_pywrap_tensorflow_internal.a failed
ld: cannot find -l_pywrap_tensorflow_internal
`
![image](https://user-images.githubusercontent.com/24863486/103599212-9538e580-4f3f-11eb-84a5-4395a180b217.png)

I would like to ask how the file “lib_pywrap_tensorflow_internal.so” is generated？

**Comment by koenvervloesem** [comment]
I'm not sure, shouldn't this be installed with Q-engineering's TensorFlow package?

Have you tried [this patch](https://github.com/tensorflow/addons/pull/2230/files)?

**Comment by genius0182** [comment]
thank you
